extensions/adobe/experience/customerJourneyManagement/ajo-entity-mixins.schema.json

```diff
@@ -208,6 +208,28 @@
               "description": "Array of Tags used in AJO entity"
             }
           }
+        },
+        "https://ns.adobe.com/experience/customerJourneyManagement/entities/directMail": {
+          "title": "Direct Mail Channel Specific Fields",
+          "type": "object",
+          "description": "Direct Mail Channel Specific Fields.",
+          "properties": {
+            "https://ns.adobe.com/experience/customerJourneyManagement/entities/routingDestination": {
+              "title": "Direct Mail Routing Destination",
+              "type": "string",
+              "description": "Direct Mail Routing Destination, non-personalized"
+            },
+            "https://ns.adobe.com/experience/customerJourneyManagement/entities/directMailFileName": {
+              "title": "Direct Mail FileName",
```

Comment:
@nikshar: Is this property a function of batch recurrence or a property of messagePublication?

As per my understanding we add entry in entityDataset at messagePublication, so if this property is at batch recurrence level we won't be able to override this value or preserve value at batch recurrence.

cc: @chetanmeh

Reply:
@sawhneyh: for DM we'll ingest this at export time
at message publication time we don't ingest this value.
this is a batch level property

Reply:
In that case, are we changing the key for this dataset also. Currently key for this datadet is a function of MessagePublicationId and VariantId.